lib/liquid/standardfilters.rb

```diff
@@ -259,6 +259,13 @@ def modulo(input, operand)
       apply_operation(input, operand, :%)
     end
 
+    def round(input, n = 0)
+      result = to_number(input).round(to_number(n))
+      result = result.to_f if result.is_a?(BigDecimal)
+      result = result.to_i if n == 0
+      result
+    end
```

Comment:
got this from the initial implementation of this filter [here](https://github.com/Shopify/liquid/commit/2239921804b40707f5e694942e8dd664d43a6912), 4 years ago 😛

Reply:
I was wondering if we could add this via a monkey patch in Measure?

Reply:
Hmm maybe inside the `Liquefier::Drop` class [here](https://github.com/yardstick/yardstick/blob/master/lib/liquefier/drop.rb#L1)? Why do we want to monkey patch though? 🤔

Reply:
As a thought experiment to do nasty nasty things with liquid filters later down the road.

Reply:
Copy that. I'll dig into this once MEAS-2842 is settled.